app/src/main/java/com/ncnf/event/create/EventCreateActivity.java

```diff
@@ -191,24 +192,24 @@ public void onClick(View v) {
                         // TODO: might change with the new event class
                         // File upload must happened after the event is saved, such that the image path
                         // contains the event's UUID
-                        FileStore file = new FileStore(Event.IMAGE_PATH, String.format(Event.IMAGE_NAME, "PLEASE_REPLACE_WITH_UUID"));
+                        FileStore file = new FileStore(Social.IMAGE_PATH, String.format(Social.IMAGE_NAME, "PLEASE_REPLACE_WITH_UUID"));
                         pictureView.setDrawingCacheEnabled(true);
                         pictureView.buildDrawingCache();
                         Bitmap bitmap = ((BitmapDrawable) pictureView.getDrawable()).getBitmap();
                         file.uploadImage(bitmap);
 
-                        //TODO: for now some fields aren't used and it only creates private event -> should be extended afterward
+                        //TODO: for now some fields aren't used and it only creates group -> should be extended afterward
                         DateAdapter date = new DateAdapter(eventDate.getYear(), eventDate.getMonthValue(), eventDate.getDayOfMonth(), eventTime.getHour(), eventTime.getMinute());
-                        PublicEvent event = new PublicEvent(user.getUuid(),
+                        Group group = new Group(user.getUuid(),
```

Comment:
The activity should be renamed if it creates a group.

Reply:
Fixed !